code/game/objects/items/robot/robot_items.dm

```diff
@@ -72,7 +72,7 @@
 	switch(mode)
 		if(CYBORG_HUGS)
 			if(M.health >= 0)
-				if(isanimal(M))
+				if(isanimal(M) && !M.holder_type)
```

Comment:
can you add a comment here explaining why `M.holder_type` is used here so people don't have to go code diving to infer purpose?

Reply:
Done!